app/src/components/Slider.tsx

```diff
@@ -0,0 +1,30 @@
+import { useState, useId } from 'react'
+
+const Slider = ({ maxRange }: { maxRange: number }) => {
+  const [sliderValue, setSliderValue] = useState(0)
+  const sliderId = useId()
+
+  const handleChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+    const newSliderValue = Number(e.target.value)
+    if (isNaN(newSliderValue)) return
```

Comment:
Instead of just returning, maybe throw an error with a custom message?

We can have an `src/errors` folder and create there the custom error files.